virttest/storage.py

```diff
@@ -364,8 +409,15 @@ def __init__(self, image, data, data_format, storage_type, **info):
             # e.g. tls-creds-psk
             self.tls_creds = info['tls_creds']
             self.reconnect_delay = info['reconnect_delay']
+        elif self.storage_type == 'curl':
+            self._password = data
+            self.cookie_auth = info['cookie_auth']
```

Comment:
Suggest both the property and parameter name to be `cookie`

Reply:
updated